src/pint/observatory/special_locations.py

```diff
@@ -70,9 +69,7 @@ def __init__(
         self.bipm_version = bipm_version
         self._bipm_clock = None
 
-        super(SpecialLocation, self).__init__(
-            name, aliases=aliases, tt2tdb_mode=tt2tdb_mode
-        )
+        super(SpecialLocation, self).__init__(name, aliases=aliases)
```

Comment:
This can just be `super().__init__(...)`.

Reply:
Is that true for 2.7?

Reply:
PINT only supports 3.6+

Reply:
I see this concern is OBE!